tgui/packages/tgui/interfaces/BarberingMenu.js

```diff
@@ -0,0 +1,196 @@
+import { useBackend, useLocalState } from '../backend';
```

Comment:
@CaioFrancisco Could you toss a license header on here? Take a look at other interfaces if you're not sure. I guess probably throw me on it as well if you want, though I will not require that as I mostly just shuffled your stuff around.

Reply:
Right, I'll do it as soon as I can.